crates/bevy_light/src/probe.rs

```diff
@@ -140,6 +140,32 @@ impl Default for GeneratedEnvironmentMapLight {
     }
 }
 
+/// Generates a cubemap of the sky produced by the atmosphere shader.
+///
+/// Attach to a `LightProbe` to capture reflections inside its volume.
+#[derive(Component, Clone)]
+pub struct AtmosphereEnvironmentMapLight {
+    /// Luminance multiplier in cd/m².
+    pub intensity: f32,
+    /// World-space rotation applied to the generated cubemap.
```

Comment:
what is this for? is there a practical application, or is it just cus you can? if i set this to like, 45 degrees around X, will the sky be sideways, or will the pixelation be sideways (but the sky still the right way)

Reply:
Not needed, let's remove this. The probe should always follow the orientation of the light being received from the atmosphere. So it should be relative to that and not relative to anything else.